test/test_arraycontext.py

```diff
@@ -1245,6 +1251,9 @@ def scale_and_orthogonalize(alpha, vel):
 
 def test_actx_compile(actx_factory):
     actx = actx_factory()
+    if isinstance(actx, CupyArrayContext):
+        pytest.skip("CupyArrayContext does not support object arrays")
+
```

Comment:
@inducer: We had briefly discussed adding support for object arrays via a wrapper (?), but I'm not sure how that should look like.